README.md

```diff
@@ -1,14 +1,15 @@
 # Cell Health Data Processing
 
+## Overview
 This is a data repository storing instructions on how to:
 
-1. Download the Cell Health Cell Painting dataset ([IDR0080](https://idr.openmicroscopy.org/webclient/?show=screen-2701))
-2. Perform segmentation
-3. Define a new single-cell focused dataset of isolated single cells
-4. Compute single cell embeddings
+0. Download the Cell Health Cell Painting dataset ([IDR0080](https://idr.openmicroscopy.org/webclient/?show=screen-2701))
+1. Perform nuclei segmentation
+2. Extract single cell embeddings
     - Using CellProfiler
     - Using DeepProfiler
-5. Process both kinds of embeddings using pycytominer
+3. Prerocess both kinds of embeddings using pycytominer
```

Comment:
```suggestion
3. Preprocess both kinds of embeddings using pycytominer
```

Reply:
It should be `Preprocess`. Thanks for catching the typo!